selenium/testcontainers/selenium/__init__.py

```diff
@@ -24,7 +25,7 @@
 }
 
 
-def get_image_name(capabilities: str) -> str:
+def get_image_name(capabilities: Dict[str, Any]) -> str:
     return IMAGES[capabilities['browserName']]
 
```

Comment:
here is the full method

Reply:
Does this mean using the old API with a `str` would have lead to an error? So we don't change the API (it was always a `dict`), we just fix the type hint?